### PR TITLE
add CloudFlare handling (based on another repo, please verify changes)

### PR DESCRIPTION
**Pull request by henderea**
I haven’t done CloudFlare Bypass changes before, but I’d really like to be able to use FlameScans again, so I copied some CloudFlare code from one of the Netsky repos I use, including the changes related to the user agent.

Please make sure my changes actually work, because I don’t know enough about this to be sure I copied everything correctly.  I may be a fairly experienced developer and familiar with TypeScript, but I barely have any experience at all with Paperback extension development.

### CODE REVIEW

File: src/FlameScans/FlameScans.ts [hunk 1]
```diff
@@ -163,6 +195,23 @@ export class FlameScans extends Source {
         return time
     }
 
+    override getCloudflareBypassRequest(): Request {
+        return createRequestObject({
+            url: FS_DOMAIN,
+            method: 'GET',
+            headers: {
+                'user-agent': userAgent,
+                'referer': `${FS_DOMAIN}.`
```
Comment (TheNetsky) [inline src/FlameScans/FlameScans.ts:204]: Should be `${FS_DOMAIN}/` not `${FS_DOMAIN}.`

**Review by TheNetsky — CHANGES_REQUESTED** [review]

**Review by TheNetsky — APPROVED** [review]
Perfect!
However, knowing CloudFlare lately if this still doesn't fix it, nothing more we can do.

File: src/FlameScans/FlameScans.ts (hunk 1 above)
Comment (henderea) [reply src/FlameScans/FlameScans.ts]: Thanks for the catch. No idea how that happened, though, because this was 95% copy-paste.

**Comment by pandeynmn** [comment]
> Perfect! However, knowing CloudFlare lately if this still doesn't fix it, nothing more we can do.

Merging the commit if there is nothing more we can fix

**Comment by henderea** [comment]
Well, there was no cloud flare bypass handling for this source before, so hopefully this will at least help.

**Comment by TheNetsky** [comment]
Increment version number.

**Comment by henderea** [comment]
Done

**Comment by TheNetsky** [comment]
Aye, good work. Thanks!

**Comment by pandeynmn** [comment]
nice

**Pull request merged**